contributors/design-proposals/pod-security-policy.md

```diff
@@ -0,0 +1,351 @@
+## Abstract
+
+PodSecurityPolicy allows cluster administrators to control the creation and validation of a security
+context for a pod and containers. The intent of PodSecurityPolicy is to protect the cluster from the
+pod and containers, not to protect a pod or containers from a user.
+
+## Motivation
+
+Administration of a multi-tenant cluster requires the ability to provide varying sets of permissions
+among the tenants, the infrastructure components, and end users of the system who may themselves be
+administrators within their own isolated namespace.
+
+Actors in a cluster may include infrastructure that is managed by administrators, infrastructure
+that is exposed to end users (builds, deployments), the isolated end user namespaces in the cluster, and
```

Comment:
This wasn't changed.  See my second commit, the first is just a move to the new name.